dns/py/runner.py

```diff
@@ -197,6 +262,10 @@ def _run_perf(self, test_case, inputs):
 
         for key, value in parser.results.items():
           results['data'][key] = value
+        results['data']['max_perfserver_cpu'] = res_usage.get()
```

Comment:
IIUC `_run_time` calculates max resource usage  and puts it into the queue every 2 second. Here we use `get`, so it returns first element of mentioned queue. So will it use max resource usage of 2 first second of runtime?

Reply:
Sorry for the very late response, I somehow thought i already replied to this. Great catch, I see I have some indenting issue. We want to add to the queue at the end of the while loop. Fixed it.